code/modules/admin/admin_verbs.dm

```diff
@@ -264,7 +264,7 @@ var/list/admin_verbs_mod = list(
 	/client/proc/cmd_admin_subtle_message,	/*send an message to somebody as a 'voice in their head'*/
 	/client/proc/cmd_admin_xeno_report,  //Allows creation of IC reports by the Queen Mother
 	/proc/release,
-	/datum/admins/proc/viewUnheardAhelps, //Why even have it as a client proc anyway? �\_("/)_/�
+	/datum/admins/proc/viewUnheardAhelps, //Why even have it as a client proc anyway? �\_("/)_/�
```

Comment:
I can't believe you'd break the shrug like this

Reply:
�\_("/)_/� blame atoms lack of UTF8 suport